hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -898,7 +898,7 @@
             - Reset the chip and repeat the first 2 steps to verify a different set of values.
             '''
       milestone: V2
-      tests: []
+      tests: ["chip_sw_lc_ctrl_hw_cfg"]
```

Comment:
Sorry, could you match with the testpoint name: `chip_sw_lc_ctrl_otp_hw_cfg`

Reply:
Sorry about that, that was my bad. I saw it but somehow forgot to change it..